test/TestHelper.cpp

```diff
@@ -486,10 +486,10 @@ int ImportTest::compareStates(State const& _stateExpect, State const& _statePost
 					TestOutputHelper::testName() + "Check State: " << a.first <<  ": incorrect storage [" << s.first << "] = " << toHex(stateStorage[s.first]) << ", expected [" << s.first << "] = " << toHex(s.second));
 
 				//Check for unexpected storage values
-				stateStorage = _stateExpect.storage(a.first);
```

Comment:
The crash happened in this assignment.  I need to dig this further.  For the time being, I just define a new variable.